testing/test_tools

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+#enable colors on CI
+export TERM=xterm-color
+
+test() {
+    local test_function=$1
+
+    echo -n "Testing ${test_function} ... "
+
+    setup
+
+    # run test function
+    output=$($test_function)
+    failed=$?
+
+    tier_down
+
+    if [ $failed != 0 ]; then
+      printRed "Failed"
+      echo ""
+      echo "------------ Test Output ------------"
+      echo "$test_function:"
+      echo "$output"
+      exit 1
+    fi
+
+    printGreen "Success"
+}
+
+setup() {
+    test_dir=$(pwd)
+
+    rm -rf temp-ww-workdir
+    mkdir $test_dir/temp-ww-workdir
+    export WW_DEFAULT_PATH=$test_dir/temp-ww-workdir
+}
+
+tier_down() {
+    cd $test_dir
+    rm -rf temp-ww-workdir
+}
```

Comment:
I believe this is a typo and should be replaced by `teardown` (no?).

Reply:
Fixed 😅